### ISSUE TITLE
Allow monoliths to be manually spawned outside of limbo/rift dungeons

### ISSUE THREAD

**Issue by Terra-B-Welch** [DimensionalDevelopment/DimDoors#170]
Would be nice for modpack creators if they wanted to use monoliths in a clever way outside of these two dimensions. Just a thought.

**Comment by BoogieMonster1O1** [comment]
Good idea.
I'll probably add this to the 1.16 port

**Comment by Terra-B-Welch** [comment]
oof I'm still on 1.12.2 :(

**Comment by Waterpicker** [comment]
> 
> 
> oof I'm still on 1.12.2 :(

SO still stuck on 1.12.2?

**Comment by Terra-B-Welch** [comment]
> 
> 
> > oof I'm still on 1.12.2 :(
> 
> SO still stuck on 1.12.2?

Yup. Though I haven't been playing minecraft as of late, why'd you dig up this issue?
